yarn.lock

```diff
@@ -2197,6 +2197,16 @@
     "@dhis2/pwa" "8.0.0"
     moment "^2.24.0"
 
+"@dhis2/app-runtime@^3":
+  version "3.2.4"
```

Comment:
Since this lock file already has `@dhis2/app-runtime@3.2.0`, we need to run `npx yarn-deduplicate yarn.lock`. Might be a good time to bump cli-app-scripts, which contains a check for this.